.github/workflows/commit-queue.yml

```diff
@@ -18,8 +18,13 @@ concurrency: ${{ github.workflow }}
 env:
   NODE_VERSION: lts/*
 
+permissions:
+  contents: read
```

Comment:
nit: we may not need `contents` permission at all for this workflow since we're using `GH_USER_TOKEN` to clone the repo – but I suppose it's easier to always have it at the top of each workflow file, it can't hurt on an open-source repo anyway.

Reply:
Yes, the `contents: read` is not strictly needed here. Setting that at the workflow level helps if a new job is added to the workflow. In that case, the new job will have `contents: read`, which is a secure-by-default setting.